packages/block-editor/src/components/image-editor/aspect-ratio-dropdown.js

```diff
@@ -54,6 +54,7 @@ export default function AspectRatioDropdown( { toggleProps } ) {
 						} }
 						value={ aspect }
 						aspectRatios={ [
+							// All ratios should be mirrored in PostFeaturedImage in @wordpress/block-library
```

Comment:
How can we share this in a single file? (can be a followup).

Reply:
The best way is probably moving the width/height/aspect ratio controls to components. They're also different values which makes it a little more tricky: the image block one needs numbers and the post featured image block needs a CSS string. Agree it's worth a follow-up.